src/main/java/network/brightspots/rcv/Tabulator.java

```diff
@@ -441,7 +442,21 @@ private void setWinningThreshold(RoundTally currentRoundTally, BigDecimal minimu
       winningThreshold = minimumVoteThreshold;
     }
 
+    if (currentRoundTally != roundTallies.get(currentRoundTally.getRoundNumber())) {
+      throw new RuntimeException("RoundTally object is not the same as the one in the map.");
+    }
```

Comment:
Why do we pass `currentRoundTally` to this method at all?

Reply:
Because we now support a per-round threshold, so currentRoundTally is used to (1) get the total number of ballots in this round, and (2) get the round number. We could replace it with two arguments (roundNum and currentRoundNumBallots) instead.

Reply:
Sorry, to clarify my question: since `roundTallies` and `roundNumber` are both instance variables, why do we need to pass args to this method to get that information? Actually, same thing for `minimumVoteThreshold` (which comes from `config`) and the `roundNumber` arg in the new `setWinningThreshold` method.

Reply:
Ah, got it -- it's not a strong preference, just a stylistic choice. The function as it stands is not required to be run on the current round.

I don't recall why I added this exception, but I assume I at some point had a bug that would have been caught by this check. As for the method signature, that is unchanged from the existing code.

I don't mind removing them, but do find it a little more readable as-is.